mllib-local/pom.xml

```diff
@@ -62,6 +62,15 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
```

Comment:
The only place we use guava is one `hashCode`. I think we should remove it from `mllib-local`'s dependencies.